sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -13,8 +13,15 @@ provider-id.setting-generator = "pluto provider-id"
 affected-services = ["kubernetes"]
 
 [metadata.settings.kubernetes.pod-infra-container-image]
-setting-generator = "schnauzer-v2 render --requires 'aws@v1' --requires 'kubernetes@v1(helpers=[pause-prefix])' --template '{{ pause-prefix settings.aws.region }}/eks/pause:3.1-eksbuild.1'"
```

Comment:
Here we're removing a metadata element, which is unusual. @jmt-lab can you please split out your new migration helper from #3828 and post a separate pull request for it?

Reply:
Opened #3958